pkg/reconciler/kubernetes/tektonconfig/extension.go

```diff
@@ -18,29 +18,72 @@ package tektonconfig
 
 import (
 	"context"
-
+	"github.com/go-logr/zapr"
+	mfc "github.com/manifestival/client-go-client"
 	mf "github.com/manifestival/manifestival"
 	"github.com/tektoncd/operator/pkg/apis/operator/v1alpha1"
 	"github.com/tektoncd/operator/pkg/client/clientset/versioned"
 	operatorclient "github.com/tektoncd/operator/pkg/client/injection/client"
 	"github.com/tektoncd/operator/pkg/reconciler/common"
 	"github.com/tektoncd/operator/pkg/reconciler/kubernetes/tektonconfig/extension"
+	"go.uber.org/zap"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"knative.dev/pkg/injection"
+	"knative.dev/pkg/logging"
+	"os"
+	"path/filepath"
 )
 
 func KubernetesExtension(ctx context.Context) common.Extension {
+	logger := logging.FromContext(ctx)
+	mfclient, err := mfc.NewClient(injection.GetConfig(ctx))
+	if err != nil {
+		logger.Fatalw("Error creating client from injected config", zap.Error(err))
+	}
+	mflogger := zapr.NewLogger(logger.Named("manifestival").Desugar())
+
+	manifest, err := mf.ManifestFrom(mf.Slice{}, mf.UseClient(mfclient), mf.UseLogger(mflogger))
+	if err != nil {
+		logger.Fatalw("Error creating initial manifest", zap.Error(err))
+	}
+
 	return kubernetesExtension{
 		operatorClientSet: operatorclient.Get(ctx),
+		manifest:          &manifest,
 	}
 }
 
 type kubernetesExtension struct {
 	operatorClientSet versioned.Interface
+	manifest          *mf.Manifest
 }
 
 func (oe kubernetesExtension) Transformers(comp v1alpha1.TektonComponent) []mf.Transformer {
 	return []mf.Transformer{}
 }
-func (oe kubernetesExtension) PreReconcile(context.Context, v1alpha1.TektonComponent) error {
+func (oe kubernetesExtension) PreReconcile(ctx context.Context, tc v1alpha1.TektonComponent) error {
```

Comment:
I see configmap for the openshift and k8s 
but this code is in k8s extension?
do we not need to add in openshift part?